src/aind_metadata_upgrader/data_description_upgrade.py

```diff
@@ -182,6 +212,11 @@ def upgrade(self, **kwargs) -> AindModel:
 
         if platform is None:
```

Comment:
You have to check the schema version - this upgrade only makes sense for schema versions prior to when we introduced  the `Platform` concept.

Reply:
should be fixed now